Source Packages/java/user/User.java

```diff
@@ -1373,6 +1345,163 @@ public boolean isOldDrupalUser() throws SQLException        {
         }
     }
     
-}
+    /**Get the username (part of email address before @) of the owner of this comment
+     * @return
+     */
+    public String getUname()
+        {
```

Comment:
I moved it down so it would be by the other getters.  When I realized it was using openID, I avoided using it.  It is used in other places (like `contactTeam()`), so I am not sure if you want to refactor it or not.